src/components/AddressSearch/AddressSearch.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useContext } from 'react';
+import DebouncedSearch from '../../util/DebouncedSearch/DebouncedSearch';
+import { getAddressAutocomplete } from "../../adapters/address-adapter"; 
+import { InputContext } from '../../App';
+
+const AddressSearch = ({ id }) => {
+    const [addressSuggestions, setAddressSuggestions] = useState([]);
+    const {inputs, handleInputChange} = useContext(InputContext);
+
+    const shouldShowDropdown = (values, currentValue) => {
+        if (!currentValue) {
+            return false;
+        }
+        if (!values || values.length == 0) {
+            return false;
+        }
+        if (values.length == 1 && values[0].address === currentValue) {
+            return false;
+        }
```

Comment:
Let's add an extra check here. If the address search input is not selected, do not show the dropdown.

Reply:
done